www.data.gov/contact.md

```diff
@@ -3,7 +3,6 @@ title: Contact
 layout: legacy-page
 scripts:
   - src: https://touchpoints.app.cloud.gov/touchpoints/1327b00b.js
-    integrity: sha256-7jQGhPOgRymVj225p6tIJtBht0oBEnXkx+ihO4oTDZk=
```

Comment:
I meant to ask about this earlier when we were testing, but no dice on getting the integrity check working?

Reply:
I pinged Ryan on the Touchpoints team with the specific error. We can add it back once we get a resolution.